src/account/AccountInfo.js

```diff
@@ -263,4 +263,42 @@ export default class AccountInfo {
             this._toProtobuf()
         ).finish();
     }
+
+    /**
+     * @returns {string}
+     */
+    toString() {
+        return JSON.stringify(this.toJSON());
+    }
+
+    // Under Development
+    // /**
+    //  * @param {string} accountInfo
+    //  * @returns {AccountInfo}
+    //  */
+    // fromString(accountInfo) {
+    //     return new AccountInfo(JSON.parse(accountInfo));
+    // }
+
+    /**
+     * @returns {{}}
```

Comment:
This should return a declared type such as https://github.com/hashgraph/hedera-sdk-js/blob/7940f062f1ca82e865ec598070eff22ce8e6e6be/src/account/AccountBalance.js#L126 https://github.com/hashgraph/hedera-sdk-js/blob/7940f062f1ca82e865ec598070eff22ce8e6e6be/src/account/AccountBalance.js#L25